hack/build/config.sh

```diff
@@ -24,6 +24,7 @@ FUNC_TEST_HTTP="cdi-func-test-file-host-http"
 FUNC_TEST_REGISTRY="cdi-func-test-registry"
 FUNC_TEST_REGISTRY_POPULATE="cdi-func-test-registry-populate"
 FUNC_TEST_REGISTRY_INIT="cdi-func-test-registry-init"
+BUILDER_TAG=${BUILDER_TAG:-kubevirt-cdi-bazel-builder@sha256:38e449bb8f5e2e3dc8ae241323c69ce37e72bd98a109d2032955dd9bd0e0a545}
```

Comment:
Okay I messed up when I gave the info, it should be:
```
BUILDER_TAG=${BUILDER_TAG:-kubevirt/kubevirt-cdi-bazel-builder@sha256:38e449bb8f5e2e3dc8ae241323c69ce37e72bd98a109d2032955dd9bd0e0a545}
```
Note the extra 'kubevirt'